script.js

```diff
@@ -0,0 +1,181 @@
+    image: 'images/speaker4.jpg',
+    imageText: 'Bill Winters',
+  },
+  {
+    name: 'Divya Seshamani',
+    title: 'Managing Partner, Greensphere',
+    info: 'She manages funds that invest in sustainable technology and projects in areas such as sustainable forestry, and energy efficiency',
+    image: 'images/speaker5.jpg',
+    imageText: 'Divya Seshamani',
+  },
+  {
+    name: 'Virginia Simmons',
+    title: 'Managing Partner, UK and Ireland, McKinsey & Company',
+    info: 'Virginia brings experience in supporting retailers and banks to drive growth and harness new technologies',
+    image: 'images/speaker6.jpg',
+    imageText: 'Virginia Simmons',
+  },
+];
+
+// conference speakers dynamic html
+
+const speakerContainerFirstHalf = document.createElement('ul');
+const speakerContainerSecondHalf = document.createElement('ul');
+const speakerUlistContainer = document.createElement('div');
+
+const moreBtn = document.createElement('button');
+const arrowDown = document.createElement('i');
+arrowDown.classList.add('fa-solid', 'fa-angle-down');
+moreBtn.style.color = '#ec5242';
+moreBtn.setAttribute('type', 'button');
+moreBtn.textContent = 'More ';
+moreBtn.classList.add('speaker-btn');
+moreBtn.appendChild(arrowDown);
+
+speakerContainerFirstHalf.classList.add('speaker-half-one-container');
+speakerContainerSecondHalf.classList.add('speaker-half-one-container', 'display-none', 'second-half');
+speakerUlistContainer.classList.add('ulist-container');
+
+for (let i = 0; i < 3; i += 1) {
+  const speakerContainer = document.querySelector('.speakers');
+  const speakerList = document.createElement('li');
+  const speakerImg = document.createElement('img');
+  const speakerInfo = document.createElement('div');
+  const speakerName = document.createElement('h4');
+  const speakerPosition = document.createElement('p');
+  const speakerStory = document.createElement('p');
+  speakerList.classList.add('speaker-list');
+  speakerImg.classList.add('speaker-img');
+  speakerInfo.classList.add('speaker-info');
+  speakerName.classList.add('speaker-name');
+  speakerStory.classList.add('speaker-story');
+  speakerPosition.classList.add('speaker-position');
+  speakerStory.classList.add('speaker-story');
+  speakerInfo.appendChild(speakerName);
+  speakerInfo.appendChild(speakerPosition);
+  speakerInfo.appendChild(speakerStory);
+  speakerList.appendChild(speakerImg);
+  speakerList.appendChild(speakerInfo);
+  speakerImg.setAttribute('src', speaker[i].image);
+  speakerImg.setAttribute('alt', speaker[i].imageText);
+  speakerName.textContent = speaker[i].name;
+  speakerPosition.textContent = speaker[i].title;
+  speakerStory.textContent = speaker[i].info;
+  speakerContainerFirstHalf.appendChild(speakerList);
+  speakerUlistContainer.appendChild(speakerContainerFirstHalf);
+  speakerContainer.appendChild(speakerUlistContainer);
+  speakerContainer.appendChild(moreBtn);
+}
+
```

Comment:
**[OPTIONAL]** - Good job implementing this section dynamically. Please look into how to use the JavaScript array [map](https://stackoverflow.com/questions/53967165/how-to-render-array-of-object-in-html-element-using-javascript-using-map-functio) method to efficiently render HTML elements dynamically in JavaScript.